core/controllers/base.py

```diff
@@ -51,8 +51,13 @@
 
 
 def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers."""
+    """Clears login cookies from the given response headers.
 
+    Args:
+        response_headers: Response Headers Object.
```

Comment:
The typeinfo should be a valid class name or standard python type; you can find it by logging the value of `type(response_headers)` or possibly by looking at the documentation for webapp2. Make sure to follow the style of the other docstrings when it comes to specifying the typeinfo.

Reply:
Hey @seanlip found it was a `webapp2.ResponseHeaders` object, modified the docstring PTAL!